2_create_project/scripts/start-stop-status

```diff
@@ -19,7 +19,8 @@ if [ -d "${ALT_CONF_DIR}" ];
 then
     CONF_DIR="${ALT_CONF_DIR}"
 fi
-ARGS="agent -config=${CONF_DIR}"
+CONF_DIR="/volume1/@appstore/nomad/nomad/conf.d/server.conf"
```

Comment:
this was blowing up on `DIR_NAME` not being set during testing

Reply:
Thing is, my config is in `/etc/nomad/server.hcl`

Not sure I'd want to ship the config with the package. I know it's not exactly great either way... What about changing the script to check either in the shipped directory or in `/etc/nomad` whichever is there.

The goal for that package was to ship the binary so DSM would play ball with it, the config was done afterwards.